src/map/itemdb.cpp

```diff
@@ -1979,6 +1979,17 @@ void itemdb_reload(void) {
 	mapit_free(iter);
 }
 
+/**
+* Reload Cash DB
+*/
+void cashdb_reload(void) {
+	struct s_mapiterator* iter;
+	struct map_session_data* sd;
+
```

Comment:
Remove `struct s_mapiterator* iter;` and `struct map_session_data* sd;` ? I got warning when compiling it.